lib/easypost/services/beta_referral_customer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# ReferralCustomer objects are User objects created from a Partner user.
+class EasyPost::Services::BetaReferralCustomer < EasyPost::Services::Service
```

Comment:
issue: in this PR, there is a `services/beta_referral_customer` file AND a `beta/referral_customer` file which I assume are probably the same? If so, we need to only have one and remove the other.

Reply:
There should only be one beta file for `referral_customer` and then one GA file for `referral_customer`, not two beta files and one GA which it looks like is the current setup.

Reply:
The other beta file is from the current non-thread-safe class. I will remove it as part of this PR. 

So we have two classes for `referral_customer `, one is called `referral_customer` for GA and the other one is called `beta_referral_customer` for Beta.

Reply:
Ah, ok then. That's fine. I think what happened was a blanket find and replace which touched these files and brought it into this PR which was a bit confusing.